sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -1984,6 +2022,20 @@ void CAFMaker::produce(art::Event& evt) noexcept {
   // rec.hdr.blind = 0;
   // rec.hdr.filt = rb::IsFiltered(evt, slices, sliceID);
 
+  // Fill the header info for the given event's spill quality info
+  if ( fHasBNBInfo && fHasNuMIInfo ) {
```

Comment:
I am all for throw an exception, that in `CAFMaker` is actually `abort()`.
I don't think this should happen, and if it does we can probably fix it if we notice.

Reply:
Fair, I can make this a loud `abort()` along with the printout.